src/test/java/net/mcreator/integration/generator/GTProcedureBlocks.java

```diff
@@ -95,6 +95,51 @@ public static void runTest(Logger LOG, String generatorName, Random random, Work
 				}
 			}
 
+			if (!procedureBlock.getRepeatingInputs().isEmpty()) {

```

Comment:
This test does no test getRepeatingAdvancedInputs, for normal inputs, there is a method getAllInputs so all inputs are tested.

Make a similar approach here

Reply:
✔️ Fixed